include/uncertainty_planning_core/uncertainty_contact_planning.hpp

```diff
@@ -141,6 +140,133 @@ class UncertaintyPolicyExecutionResult
   }
 };
 
+#if UNCERTAINTY_PLANNING_CORE__SUPPORTED_ROS_VERSION == 2
+class ExecutionTimeLimit
+{
+public:
+  ExecutionTimeLimit(const double seconds, rclcpp::Clock::SharedPtr clock)
+    : seconds_(seconds), clock_(clock)
+  {
+    if (seconds_ < 0.0)
+    {
+      throw std::invalid_argument("Time limit cannot be less than zero seconds");
+    }
+    if (!clock_)
+    {
+      throw std::invalid_argument("No valid clock supplied");
+    }
+  }
+
+  explicit ExecutionTimeLimit(const double seconds)
+    : ExecutionTimeLimit(seconds, std::make_shared<rclcpp::Clock>())
+  {
```

Comment:
In cases like this I try to have the brackets at the end of the line
```
explicit ExecutionTimeLimit(const double seconds)
    : ExecutionTimeLimit(seconds, std::make_shared<rclcpp::Clock>()) {}
```

Reply:
See https://github.com/calderpg/uncertainty_planning_core/pull/3/commits/cf8ce915f230816208852918241ed1d6227c0d35.